src/lib.rs

```diff
@@ -268,18 +272,18 @@ impl<F: io::Write> io::Write for File<F> {
 ///
 /// ```
 /// extern crate futures;
-/// extern crate tokio_core;
+/// extern crate tokio_io;
 /// # extern crate tokio_file_unix;
 ///
 /// use futures::Stream;
-/// use tokio_core::io::Io;
+/// use tokio_io::io::Io;
```

Comment:
`tokio_io::io::Io` does not exist.  It appears to have moved to [`tokio_io::AsyncRead::framed`](https://docs.rs/tokio-io/0.1/tokio_io/trait.AsyncRead.html).

This also means that `tokio_io::{AsyncRead, AsyncWrite}` have to be implemented for `File<F>`, as `tokio_io` no longer uses the `std::io::{Read, Write}` traits.